<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoWindow</class>
 <widget class="QWidget" name="TodoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QPushButton" name="signOutpushButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>520</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign Out</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1011</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Projects</string>
     </attribute>
     <widget class="QLabel" name="projectnamelabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Project name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="projectnamelineEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>140</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="textlabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Project description:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textlineEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>170</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="projectSavepushButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="projectUpdatepushButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="projectDeletepushButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>220</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>30</y>
        <width>301</width>
        <height>401</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Labels</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>40</y>
        <width>201</width>
        <height>401</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="labelnamelabel">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>150</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Label name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="labelnamelineEdit">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>150</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="labelSavepushButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="labelUpdatepushButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="labelDeletepushButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tasks</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>601</width>
        <height>411</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Deadline</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Project</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Label</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="tasknamelabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="taskdesclabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
     <widget class="QLabel" name="projectnamelabel_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Deadline:</string>
      </property>
     </widget>
     <widget class="QLabel" name="projectnamelabel_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Project:</string>
      </property>
     </widget>
     <widget class="QLabel" name="projectnamelabel_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Label:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="tasknamelineEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="taskDesclineEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="taskDeadlinelineEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="taskprojectlineEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="tasklabellineEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>260</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="taskSavepushButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="taskUpdatepushButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="taskDeletepushButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
